src/components/LayoutHeader/Nav/styles.module.css

```diff
@@ -20,111 +20,158 @@
 
 .linkItem {
   position: relative;
-  margin: 0 15px;
+  margin-left: 4px;
 
-  & + &.icon {
-    margin-left: 15px;
+  &.socialIconLinkItem + &.socialIconLinkItem {
+    margin: 0 16px 0 0;
   }
 
-  &.icon {
-    margin: 0 5px;
+  &:first-of-type {
+    margin: 0;
+  }
+
+  @media (--xl-scr) {
+    margin-left: 12px;
+
+    &.socialIconLinkItem + &.socialIconLinkItem {
+      margin: 0 24px 0 0;
+    }
+  }
+
+  @media (--md-scr) {
+    margin-left: 0;
 
-    &:first-of-type {
-      margin-left: 15px;
+    &:first-child {
+      display: none;
+    }
+  }
+
+  @media (--sm-scr) {
+    margin-left: 4px;
+
+    &:nth-child(5),
```

Comment:
I used CSS to take off the nav items, but we could also use JS to check for screen overflow and take off the items with JS.